x/claim/keeper/grpc_query_airdrop_supply_test.go

```diff
@@ -1,22 +1,26 @@
 package keeper_test
 
 import (
+	"github.com/tendermint/spn/testutil/sample"
 	"testing"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/stretchr/testify/require"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
 
-	keepertest "github.com/tendermint/spn/testutil/keeper"
+	testkeeper "github.com/tendermint/spn/testutil/keeper"
 	"github.com/tendermint/spn/testutil/nullify"
 	"github.com/tendermint/spn/x/claim/types"
 )
 
 func TestAirdropSupplyQuery(t *testing.T) {
-	keeper, ctx := keepertest.ClaimKeeper(t)
+	ctx, tk, _ := testkeeper.NewTestSetup(t)
+
 	wctx := sdk.WrapSDKContext(ctx)
-	item := createTestAirdropSupply(keeper, ctx)
+	sampleSupply := sample.Coin(r)
+	tk.ClaimKeeper.SetAirdropSupply(ctx, sampleSupply)
+
 	for _, tc := range []struct {
 		desc     string
 		request  *types.QueryGetAirdropSupplyRequest
```

Comment:
We can compare only the `AirdropSupply`

```suggestion
		request  types.AirdropSupply
```

Reply:
I don't get your comment

The method is request -> response, therefore we assert the expected response with the actual one
There is one test case with `nil` request

Reply:
yes, but we don't need the entire object to compare. The test case can be simpler

```go
		{
			desc:     "First",
			request:  &types.QueryGetAirdropSupplyRequest{},
			supplyResponse: sampleSupply,
		},
```

Reply:
and we can compare only the supply

```go
				require.Equal(t,
					nullify.Fill(tc.supplyResponse),
					nullify.Fill(response. AirdropSupply),
				)
```

Reply:
I see.
I find it more visual to use response since it's the format we have for all grpc tests